komodo/__init__.py

```diff
@@ -1,12 +1,14 @@
-"""Komodo software distribution build system."""
+from pkg_resources import DistributionNotFound, get_distribution
 
-__version__ = "1.0"
-__author__ = "Software Innovation Bergen, Statoil ASA"
-
-__copyright__ = "Copyright 2017, Statoil ASA"
-__license__ = "GNU General Public License, version 3 or any later version"
-
-__credits__ = __author__
-__maintainer__ = __author__
-__email__ = "fg_gpl@statoil.com"
-__status__ = "Production"
+try:
+    VERSION = get_distribution(__name__).version
+except DistributionNotFound:
+    try:
+        from ._version import version as VERSION
+    except ImportError:
+        raise ImportError(
+            "Failed to find (autogenerated) _version.py. "
+            "This might be because you are installing from GitHub's tarballs, "
+            "use the PyPI ones."
```

Comment:
This comment might lead people to https://pypi.org/project/komodo/ which is not exacly not we want..

Reply:
That's a good point @berland - missed that comment. Just to mention use `pip install . ` should suffice?

Reply:
Yes, good point; removed references to GitHub and PyPI... and trying to figure out something weird in my installation. Not exactly sure yet what someone will get when using `pip install .` on a cloned repo.

Reply:
In fact, `pip install .` works fine. But `komodo.__version__` won't do anything interesting until we start making release tags, if we ever do.

Anyway, leaving the message as "Failed to find autogenerated _version.py.".